core/services/ocr2/plugins/ocr2keeper/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/pkg/errors"
+)
+
+// PluginConfig contains custom arguments for the OCR2Keeper plugin.
+type PluginConfig struct {
+	LinkEthFeedAddress string   `json:"linkEthFeedAddress"`
+	ConfirmationDelays []uint32 `json:"confirmationDelays"`
+	LookbackBlocks     int64    `json:"lookbackBlocks"`
```

Comment:
I'm wondering if there's a better way to get a shared source of pseudo-randomness now that the nodes can communicate directly. I think this method works fine, but has created some strange edge cases for us. wdyt?